src/Components/booklist.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from './book';
```

Comment:
- [ ] As mentioned by the previous **[reviewer](https://github.com/Becky449/bookstore/pull/5)** to fix your imported files, kindly make sure you did it for all files, as right now  I am getting errors when trying to run the optimized production, all of them are caused by typing errors. And this is why you get the linters errors. Once you fix this you can remove the `/* eslint-disable */` comment
```JavaScript
// Bad
import Book from './book';

// Good
import Book from './Book';
```

This issue took me a long time to run your Application